### DISCUSSION TITLE
anyway in the format syntax to specify a CLIPBOARD content output?

### DISCUSSION THREAD

**Discussion by zeltak** [chhoumann/quickadd#529] [General]
Hi

I cant seem to find a way  in the format syntax to specify a CLIPBOARD content output? would love to have something like this

```
#task {{VALUE:task name}} 📅 {{VDATE:due date,YYYY-MM-DD}} {{VALUE:tags}}  {{CLIPBOARD}}

```

thx a ton!

Z

**Comment by FelipeRearden** [comment]
Hello @zeltak and @chhoumann 

Trying to help here with my limited knowledge

When I wanna use the clipboard, I use this inline-script:

```
\`\`\`js quickadd
return await this.quickAddApi.utility.getClipboard();
\`\`\`
```

Applying to your example:
![image](https://github.com/chhoumann/quickadd/assets/85180696/1e281259-3397-49d1-bf12-7bca412642c6)

I hope it helps 🙏

**Comment by zeltak** [comment]
@FelipeRearden 

thx so much this is so awesome!!!

I **REALLY** should learns some programming...maybe when the kids grow up haha :) :)

Z

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that. And thanks @FelipeRearden for the inline-script workaround, which was the right call at the time.

There's a native token for this now: `{{CLIPBOARD}}` has been part of the format syntax since QuickAdd 2.0.0, so your example works verbatim, no script needed:

```
#task {{VALUE:task name}} 📅 {{VDATE:due date,YYYY-MM-DD}} {{VALUE:tags}} {{CLIPBOARD}}
```

Bonus: in Capture content, if the clipboard holds an image instead of text, QuickAdd saves it to your attachment folder and embeds it.

Docs: https://quickadd.obsidian.guide/docs/FormatSyntax#clipboard

Closing as resolved.
